src/main/java/org/assertj/core/api/AbstractLongAdderAssert.java

```diff
@@ -0,0 +1,552 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2020 the original author or authors.
+ */
+package org.assertj.core.api;
+
+import org.assertj.core.data.Offset;
+import org.assertj.core.data.Percentage;
+import org.assertj.core.internal.ComparatorBasedComparisonStrategy;
+import org.assertj.core.internal.Longs;
+import org.assertj.core.util.CheckReturnValue;
+import org.assertj.core.util.VisibleForTesting;
+
+import java.util.Comparator;
+import java.util.concurrent.atomic.LongAdder;
+
+import static org.assertj.core.error.ShouldHaveValue.shouldHaveValue;
+import static org.assertj.core.error.ShouldNotContainValue.shouldNotContainValue;
+
+/**
+ * Base class for all implementations of assertions for {@link LongAdder}s.
+ *
+ * @param <SELF> the "self" type of this assertion class. Please read &quot;<a href="http://bit.ly/1IZIRcY"
+ *               target="_blank">Emulating 'self types' using Java Generics to simplify fluent API implementation</a>&quot;
+ *               for more details.
+ *
+ * @author Grzegorz Piwowarek
+ */
+class AbstractLongAdderAssert<SELF extends AbstractLongAdderAssert<SELF>> extends AbstractAssert<LongAdderAssert, LongAdder> implements NumberAssert<LongAdderAssert, Long> {
```

Comment:
I think this should be:
```suggestion
public class AbstractLongAdderAssert<SELF extends AbstractLongAdderAssert<SELF>> extends AbstractAssert<SELF, LongAdder> implements NumberAssert<SELF, Long> {
```

Reply:
Yes, good catch!